packages/apps/human-app/frontend/src/api/fetcher.ts

```diff
@@ -99,10 +102,18 @@ export function createFetcher(defaultFetcherConfig?: {
       : fetcherOptions.options;
     if (fetcherOptions.authenticated) {
       fetcherOptionsWithDefaults = appendHeader(fetcherOptionsWithDefaults, {
-        Authorization: `Bearer ${browserAuthProvider.getAccessToken() ?? ''}`,
+        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```

Comment:
Wouldn't it be better if we change the rule globally in order to allow not only strings, but also booleans, numbers and null/undefined?

Reply:
@KacperKoza343 could you please do it in this PR?

Reply:
done